File: Reactivities.Server/API/Profiles/Models/ProfileApiModel.cs
using API.Common.Models;

namespace API.Profiles.Models
{
    public class ProfileApiModel : BaseApiModel
    {
        public string Email { get; set; }

        public string Bio { get; set; }

        public string DisplayName { get; set; }

        public int FollowersCount { get; set; }
        
        public int FollowingsCount { get; set; }

        public bool Following { get; set; }
        public string Username { get; set; }

        public string PictureUrl { get; set; }
    }
}

File: client/src/autogenerated/a-p-i/profiles/profile-api-model.ts
import { BaseApiModel } from '../common/base-api-model';

export class ProfileApiModel extends BaseApiModel {
    email!: string;
    bio!: string;
    displayName!: string;
    followersCount!: number;
    followingsCount!: number;
    following!: boolean;
    username!: string;
    pictureUrl!: string;
}
